### PR TITLE
Fixes Armory Crashes

### LINKED ISSUE #11037: [META] Crashing near science
**Problem Description**:
Me and others have crashed multiple times now when approaching science on metastation

**What did you expect to happen**: 
Not to crash

**What happened instead**:
CRASH

**Why is this bad/What are the consequences:**
It crashes you and takes you out of the game for 30+ seconds (yes ads that don't load)

**Steps to reproduce the problem**:
Walk near science. It happend to me in the hallway to escape

### LINKED ISSUE #11142: Crash Tiles at the Armory
**Problem Description**:
The crashing hallway tiles on metastation seem to now happen on the Cyberiad, though in the brig armory area. The small spot with the vendor and the secure armory itself. Once again random where exactly it happens. Happened on two separate rounds, does not always happen where it did previously or even at all.

**What did you expect to happen**: 
I expected to not crash when entering the armory.

**What happened instead**:
Myself and 3-4 other officers crashed when entering the armory.

**Why is this bad/What are the consequences:**
Number of reasons. Being chased by an antag, trying to gear up for a threat, generally existing to go recharge your gear.

**Steps to reproduce the problem**:
1. Be sec.
2. Go into the armory room.
3. Walk around on the tiles until you possibly crash.

**When did the problem start happening**:
About a few days ago is when I noticed it when I started playing security again.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was anything else out of the ordinary happening?)**:
Nothing relevant.

### PR DESCRIPTION
**Pull request by Ty-Omaha**
## What Does This PR Do
Fixes the crashes that sometimes happen at the armory by removing the bullet icons that appear ontop of shooting targets. That explains why it only happens sometimes and not all the time.

Similar to #9527 where the Art Easel Canvas was making people crash.

## Why It's Good For The Game
So people stop crashing

Fixes #11142 
Fixes #11037
:cl:
fix: Armory crashing should be fixed, shooting range targets will no longer have bulletholes appear when shot.
/ :cl:

### CODE REVIEW

**Comment by farie82** [comment]
On meta station a similar issue occurs towards escape near science. I think there are also targets there...

Know why this is causing crashes or is it just byond being weird again?

**Comment by Spacemanspark** [comment]
Awww man, even if it's small I'm going to miss this feature

**Comment by Ty-Omaha** [comment]
>On meta station a similar issue occurs towards escape near science. I think there are also targets there...
Know why this is causing crashes or is it just byond being weird again?

This will fix that too, and I don't know I think Byond just can't handle drawing things

**Comment by AzuleUtama** [comment]
Flashbacks to when a traitor emagged into the armory and got dc'd, followed by 3 officers going after him and then also dc'ing right next to him.

Thank god it's fixed.

**Comment by farie82** [comment]
Fixes: #11037 and fixes: #11142

File: code/game/objects/items/shooting_range.dm [hunk 1]
```diff
@@ -79,102 +77,3 @@
 	icon_state = "target_q"
 	desc = "A shooting target that looks like a xenomorphic alien."
 	hp = 2350 // alium onest too kinda
-
-/obj/item/target/bullet_act(var/obj/item/projectile/Proj)
```
Comment (Fox-McCloud) [inline code/game/objects/items/shooting_range.dm]: Improve, don't remove. We have mutable appearances for a reason, these days.

Comment (Ty-Omaha) [reply code/game/objects/items/shooting_range.dm]: no clue how to use them, game stability is more important than tiny bullet holes on a target

Comment (taukausanake) [reply code/game/objects/items/shooting_range.dm]: /TG/ fixed whatever was causing their canvases to crash players. This is related to it (at least I think it is) so shouldn't we be looking at what they did?

Comment (Ty-Omaha) [reply code/game/objects/items/shooting_range.dm]: @taukausanake you're free to put up a PR

**Comment by Fox-McCloud** [comment]
Closing in favor of: https://github.com/ParadiseSS13/Paradise/pull/13618

**Pull request closed without merging**